index.html

```diff
@@ -491,23 +491,32 @@ <h3 class="most-popular__title">Lo más popular</h3>
       <h4 class="popular-brands__title">Marcas populares</h4>
       <div class="popular-brands__background">
         <div class="popular-brands__icons">
-          <div class="popular-brands__carousel js-popular-brands-carousel ">
-            <a href="#" class="popular-brands__img">
-              <img src="image/logo-1.png" alt="Logo 1">
-            </a>
-            <a href="#" class="popular-brands__img">
-              <img src="image/logo-2.png" alt="Logo 2">
-            </a>
-            <a href="#" class="popular-brands__img">
-              <img src="image/logo-3.png" alt="Logo 3">
-            </a>
-            <a href="#" class="popular-brands__img">
-              <img src="image/logo-4.png" alt="Logo 4">
-            </a>
-            <a href="#" class="popular-brands__img">
-              <img src="image/logo-5.png" alt="Logo 5">
-            </a>
-          </div>
+          <ul class="popular-brands__carousel js-popular-brands-carousel">
+            <li>
+              <a href="#" class="popular-brands__img" aria-label="visitar proveedor">
```

Comment:
Seria bueno incluir el nombre del proveedor
```suggestion
              <a href="#" class="popular-brands__img" aria-label="visitar proveedor juanito">
```